.circleci/config.yml

```diff
@@ -188,7 +188,7 @@ jobs:
       - checkout
       - attach_workspace:
           at: .
-      - install-gems
+      - revenuecat/install-rubydocker-dependencies
```

Comment:
Hmm I might be wrong but do we also need to update the android jobs? Not sure if the `android-machine` uses mac or linux, but if it's using linux, that should also use `install-rubydocker-dependencies`

Reply:
I think you're correct! The problem comes from the Apple executor storing the caches then the Android machine trying to restore them, not because of this change I just did 😮‍💨

Reply:
Also, we should update to just be `install-docker-dependencies` right?

Reply:
yeah, I think that or even just rename to `install-gem-mac-dependencies` and `install-gem-unix-dependencies` or something like that, since the difference is due to the OS.

Reply:
I can update that and move the install-gem-mac-dependencies to the orb. Also adding a version for invalidation as a parameter of the job will be convenient